src/ui/wxWidgets/StrengthMeter.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Initial version created as 'StrengthMeter.h'
+ * by rafaelx on 2025-05-01.
+ *
+ * Copyright (c) 2019-2025 Rony Shapiro <ronys@pwsafe.org>.
+ * All rights reserved. Use of the code is allowed under the
+ * Artistic License 2.0 terms, as specified in the LICENSE file
+ * distributed with this code, or available from
+ * http://www.opensource.org/licenses/artistic-license-2.0.php
+ */
+
+#ifndef STRENGTH_METER_H
+#define STRENGTH_METER_H
+
+#include <wx/wx.h>
+
+class StrengthMeter : public wxControl {
+public:
+  static const int STRENGTH_MIN;
```

Comment:
Nit: why not assign the values here, since these are constants?

Reply:
Changed as proposed. - Thanks for the review.